staging/src/kubevirt.io/client-go/kubecli/kubevirt.go

```diff
@@ -268,15 +268,9 @@ type VirtualMachineInstanceInterface interface {
 }
 
 type ReplicaSetInterface interface {
-	Get(ctx context.Context, name string, options metav1.GetOptions) (*v1.VirtualMachineInstanceReplicaSet, error)
-	List(ctx context.Context, opts metav1.ListOptions) (*v1.VirtualMachineInstanceReplicaSetList, error)
-	Create(ctx context.Context, replicaset *v1.VirtualMachineInstanceReplicaSet, opts metav1.CreateOptions) (*v1.VirtualMachineInstanceReplicaSet, error)
-	Update(ctx context.Context, virtualMachineInstanceReplicaSet *v1.VirtualMachineInstanceReplicaSet, opts metav1.UpdateOptions) (*v1.VirtualMachineInstanceReplicaSet, error)
-	Delete(ctx context.Context, name string, options metav1.DeleteOptions) error
+	kvcorev1.VirtualMachineInstanceReplicaSetInterface
 	GetScale(replicaSetName string, options metav1.GetOptions) (*autov1.Scale, error)
 	UpdateScale(replicaSetName string, scale *autov1.Scale) (*autov1.Scale, error)
```

Comment:
Is there a follow-up planned for adding context to these methods?

Reply:
I will add a commit here! Thank you!

Reply:
Added 2 commits for the context addition to updatescale and getscale.
PTAL